dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2

```diff
@@ -104,6 +104,9 @@ router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }}
     network {{ get_ipv6_loopback_address(LOOPBACK_INTERFACE, "Loopback0") | ip }}/128
 {% else %}
     network {{ get_ipv6_loopback_address(LOOPBACK_INTERFACE, "Loopback0") | ip }}/64
+{% if DEVICE_METADATA['localhost']['switch_type'] == 'voq' or DEVICE_METADATA['localhost']['switch_type'] == 'chassis-packet' %}
```

Comment:
In multi asic linecard, both asics will have same Loopback0. how will this change effect in this case?

Reply:
connected routes in the local table have the highest priority